Places/kumari.md

```diff
@@ -0,0 +1,7 @@
+#### Places to eat
+ - Best bakery
+ - India Cafe
```

Comment:
Wow! Great to know that. Masala Dosa and Onion Vada are the great items to try :smile: